### DISCUSSION TITLE
无法登录客户端

### DISCUSSION THREAD

**Discussion by Y23F5** [natfrp/wiki#322] [使用问题] [answered]
![image](https://github.com/natfrp/wiki/assets/50901063/570fa165-102b-446c-af84-7dc295584ada)
在重新安装/卸载守护进程服务/管理员身份运行后，均弹出此报错，无法登录客户端

**Comment by 1235689hhh** [comment]
同问题,我是在自动登陆上不去之后手动删除了位于appdate里面关于账号信息的文件夹后手动登陆触发的问题,由于我人在湖南,初步猜测
1.在被dns污染后区域登陆皆会出现此现象,但是如果有已登陆账号自动登陆就会导致警报不触发
2.在删除为于c盘appdate下的文件夹后app读取资料出错报错
目前更怀疑是1

[a comment by an automated account is not shown]
